### ISSUE TITLE
'Client' object has no attribute 'client'

### ISSUE THREAD

**Issue by WhistleMaster** [Minims/SomfyProtect2MQTT#50]
Hi !

Thanks for the great work on that project ! I'm using latest version 0.2.0 with docker and everything seems to work except that I have to following error in the log when, as an example, arming / disarming the alarm or opening/closing the camera cover:

```
2022-06-06 07:35:52,179 [WARNING] [business.mqtt:40] Error while refreshing Camera: 'Client' object has no attribute 'client'
2022-06-06 07:35:05,358 [WARNING] [business.mqtt:40] Error while refreshing Camera: 'Client' object has no attribute 'client'
2022-06-06 07:36:29,454 [WARNING] [business.mqtt:40] Error while refreshing Camera: 'Client' object has no attribute 'client'
2022-06-06 07:36:46,591 [WARNING] [business.mqtt:40] Error while refreshing Camera: 'Client' object has no attribute 'client'
2022-06-06 07:37:09,270 [WARNING] [business.mqtt:58] Error while refreshing site <<redacted>>: 'Client' object has no attribute 'client'
2022-06-06 07:37:16,256 [WARNING] [business.mqtt:58] Error while refreshing site <<redacted>>: 'Client' object has no attribute 'client'
```

Kind regards,

WM

**Comment by Minims** [comment]
Hello, 

Ok I can reproduce, I need to debug a bit, will fix in next release.

**Comment by Minims** [comment]
Must be fixed in 0.2.1
